README.md

```diff
@@ -518,6 +518,64 @@ knife azurerm server create
   -c ~/.chef/knife.rb
 ```
 
+For Redhat:
+
+```
+knife azurerm server create
+  --azure-resource-group-name MyResourceGrpName
+  --azure-vm-name MyNewVMName
+  --azure-service-location 'WEST US'
+  --azure-image-os-type redhat
```

Comment:
It will be better to use `rhel` as the value of `--azure-image-os-type` because `redhat` may have other products also apart Linux OS.